backend/support/cog_over.py

```diff
@@ -123,7 +123,7 @@ def lookup_baseline(ein, uei, dbkey):
 def get_2019_gen(ein, dbkey):
     gens = CensusGen19.objects.annotate(
         amt=Cast("totfedexpend", output_field=BigIntegerField())
-    ).filter(Q(ein=ein), Q(dbkey=dbkey) | Q(dbkey=None))
```

Comment:
This one too is incorrect for the same reason as the first one.

Reply:
This is correct based on the findings and also the discussions in slack and meetings.
https://docs.google.com/document/d/1SaBRMdr7HsHqHuDZ98oxsJOvQGQPlYjEmk7Zhky8mXM/edit